config/default.json

```diff
@@ -5,7 +5,7 @@
     "atnaUrl": "mongodb://localhost/openhim",
     "connectionOptions": {},
     "openHIMApiReadPreference": "primaryPreffered",
-    "openHIMApiWriteConcern": "1",
+    "openHIMApiWriteConcern": "majority",
```

Comment:
I think this can be changed back to **1**, just ensure that it is a integer and not a string